File: POTD/May'24/Easy/C++/Reverse_prefix_of_word.cpp
class Solution {
public:
    string reversePrefix(string word, char ch) {
        int n = word.find(ch);
        if(n != string::npos){
            int i = 0,j = n;
            while(i < j){
                swap(word[i],word[j]);
                i++;j--;
            }
        }
        return word;
    }
};

File: POTD/May'24/Python/Easy/Reverse_prefix_of_word.py
class Solution:
    def reversePrefix(self, word: str, ch: str) -> str:
        n = word.find(ch)
        i, j = 0,n
        arr = list(word)
        if n != -1:
            while i < j:
                arr[i],arr[j] = arr[j],arr[i]
                i += 1
                j -= 1
        return ''.join(arr)